implementation/prometheus/src/main/java/net/opentsdb/servlet/resources/PromQLResource.java

```diff
@@ -245,8 +248,9 @@ private Response handleQuery(final ServletConfig servlet_config,
           .build();
     }
     
-    long query_timeout = (long) servlet_config.getServletContext()
+    int async_timeout = (Integer) servlet_config.getServletContext()
```

Comment:
cast it to `int` instead of `Integer`

Reply:
It's already auto boxed to an `Integer` due to being in the servlet map so it's ok to keep it like this. The cast from `Integer` to `long` was broken though.